DependencyInjection/Configuration.php

```diff
@@ -50,9 +50,17 @@
  *   - [bubble]: bool, defaults to true
  *
  * - gelf:
- *   - publisher: {id: ...} or {hostname: ..., port: ..., chunk_size: ...}
- *   - [level]: level name or int value, defaults to DEBUG
- *   - [bubble]: bool, defaults to true
+ * - publisher: (one of the following configurations)
+ *   # Option 1: Service-based configuration
+ *   - id: string, service id of a publisher implementation
+ *
+ *   # Option 2: Direct connection configuration
+ *   - hostname: string, server hostname
+ *   - [port]: int, server port (default: 12201)
+ *   - [chunk_size]: int, UDP packet size (default: 1420)
+ *   - [encoder]: string, encoding format ('json' or 'compressed_json')
+ * - [level]: level name or int value, defaults to DEBUG
+ * - [bubble]: bool, defaults to true
```

Comment:
The indentation is wrong:

```suggestion
 *   - publisher: (one of the following configurations)
 *     # Option 1: Service-based configuration
 *     - id: string, service id of a publisher implementation
 *
 *     # Option 2: Direct connection configuration
 *     - hostname: string, server hostname
 *     - [port]: int, server port (default: 12201)
 *     - [chunk_size]: int, UDP packet size (default: 1420)
 *     - [encoder]: string, encoding format ('json' or 'compressed_json')
 *   - [level]: level name or int value, defaults to DEBUG
 *   - [bubble]: bool, defaults to true
```